kotlin-react-router-dom/src/main/kotlin/react/router/dom/imports.kt

```diff
@@ -1,19 +1,24 @@
-@file:JsModule("react-router-dom")
+@file:Suppress("PropertyName")
 
 package react.router.dom
 
 import react.RClass
 import react.RProps
 
-external fun useHistory(): RouteResultHistory
+external interface ReactRouterDom {
+    fun useHistory(): RouteResultHistory
+    fun useLocation(): RouteResultLocation
+    fun useParams(): dynamic
+    fun <T : RProps> useRouteMatch(options: dynamic): RouteResultMatch<T>
+    fun <T : RProps> withRouter(component: dynamic): RClass<T>
+    val HashRouter: RClass<RProps>
+    val BrowserRouter: RClass<RProps>
+    val Switch: RClass<RProps>
+    val Route: RClass<dynamic>
+    val Link: RClass<LinkProps>
+    val NavLink: RClass<dynamic>
+    val Redirect : RClass<RedirectProps>
+}
 
-external fun useLocation(): RouteResultLocation
-
-@JsName("useParams")
-external fun rawUseParams(): dynamic
-
-@JsName("useRouteMatch")
-external fun <T : RProps> rawUseRouteMatch(options: dynamic): RouteResultMatch<T>
-
-@JsName("withRouter")
-external fun <T : RProps> rawWithRouter(component: dynamic): RClass<T>
+@JsModule("react-router-dom")
+external val ReactRouterDomModule: ReactRouterDom
```

Comment:
Do we really need it?
Could we use
```
@JsModule("react-router-dom")
external object ReactRouterDom {
...
}
```
Or do you see some caveats?

Reply:
As you pointed out, Object was sufficient😇